app/controllers/admin/courses_controller.rb

```diff
@@ -0,0 +1,15 @@
+module Admin
+  class CoursesController < ApplicationController
+    before_action { authorize! }
+
+    respond_to :html
+    layout 'assignment_content'
+
+    def index
+      respond_to do |format|
+        format.html { page_not_found }
```

Comment:
Placed this here since it may be easy to accidentally get to this page (the edit/create courses view will have their own view and would thus have a url pattern of `:markus_url/admin/courses/:id`. Not sure though if it is needed.